consul-connect/provision.courseData.sh

```diff
@@ -0,0 +1,72 @@
+cat << 'EOFSRSLY' > /tmp/provision.sh
+#! /bin/bash
+
+log() {
+  echo $(date) - ${@}
+}
+
+finish() {
+  touch /provision_complete
+  log "Complete!  Move on to the next step."
+}
+
+#############################
+
+log "Install prerequisites"
+# apt-get install -y apt-utils > /dev/null
+apt-get install -y unzip curl jq > /dev/null
```

Comment:
`curl` should be available in Ubuntu already, and maybe `unzip`. If we need `jq` we can add it to the `hashicorp-hashistack` image.